docs/source/markdown/podman-auto-update.1.md

```diff
@@ -26,6 +26,12 @@ If an image ID was used, Podman would not know which image to check/pull anymore
 Moreover, the systemd units are expected to be generated with `podman-generate-systemd --new`, or similar units that create new containers in order to run the updated images.
 Systemd units that start and stop a container cannot run a new image.
 
+
+### Systemd Unit and Timer
+
+Podman ships with a `podman-auto-update.service` systemd unit.  This unit is triggered daily by the `podman-auto-update.timer` systemd timer.  The timer can be altered for custom time-based updates if desired.  The unit can further be invoked by other systemd units (e.g., via the dependency tree) or manually via `systemctl start podman-auto-update.service`.
```

Comment:
AFAIK daily in systemd timer context means daily at midnight. Can you add this information because I want to know when exactly it is triggered. I hate statements such as daily, weekly monthly, etc...

Reply:
Done